.mci.yml

```diff
@@ -531,6 +531,35 @@ tasks:
             export INSTALL_VERSION=${mongocxx_version_finder_sh}
             cmd.exe /c .\\.evergreen\\uninstall_check_windows.cmd
 
+    - name: debian-package-build
+      commands:
+      - func: "setup"
+      - command: expansions.update
```

Comment:
Is the `mongocxx_version_finder_sh` expansion unused in this task?

Reply:
Yes, though I failed to notice this until after I pushed the PR.  I have an unpushed commit that removes this.  When I update the PR, this will be removed.